kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -7095,20 +7108,20 @@ def _start_vm(self, cmd):
             logger.warn(linux.get_exception_stacktrace())
 
             # c.f. https://access.redhat.com/solutions/2735671
-            if "org.fedoraproject.FirewallD1 was not provided" in str(e.message):
+            if "org.fedoraproject.FirewallD1 was not provided" in str(e):
                 _stop_world()  # to trigger libvirtd restart
                 raise kvmagent.KvmError(
                     'unable to start vm[uuid:%s, name:%s], libvirt error: %s' % (
                     cmd.vmInstanceUuid, cmd.vmName, str(e)))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)